src/app/(proper_react)/redesign/(authenticated)/user/(dashboard)/settings/page.tsx

```diff
@@ -24,6 +24,7 @@ export default async function SettingsPage() {
   const monthlySubscriptionUrl = getPremiumSubscriptionUrl({ type: "monthly" });
   const yearlySubscriptionUrl = getPremiumSubscriptionUrl({ type: "yearly" });
   const fxaSettingsUrl = process.env.FXA_SETTINGS_URL!;
+  const fxaSubscriptionsUrl = process.env.FXA_SUBSCRIPTIONS_URL!;
```

Comment:
Q: Does this need `NEXT_PUBLIC_` prefix?

```sh
git grep -in "FXA_SUBSCRIPTIONS_URL" | cat

.env-dist:136:FXA_SUBSCRIPTIONS_URL=https://accounts.stage.mozaws.net/subscriptions
src/app/(proper_react)/redesign/(authenticated)/user/(dashboard)/settings/page.tsx:27:  const fxaSubscriptionsUrl = process.env.FXA_SUBSCRIPTIONS_URL!;
```

Reply:
https://github.com/mozilla/blurts-server/pull/4005